PF.Api/Services/PensionCalculator.cs

```diff
@@ -29,13 +36,34 @@ public double Calculate(Person person)
 
         public bool CanUseStandardPension(Person person)
         {
-            // TODO: check if person can have default pension based on experience
-            throw new NotImplementedException();
+            var age = GetDifferenceInYears(DateTime.Today, person.DateOfBirth);
+
+            if (MALE.Equals(person.Sex, StringComparison.OrdinalIgnoreCase) && age < _settings.MinAgeMale || age < _settings.MinAgeFemale)
+                    return false;
+
+            var exp = person.Experiences?.Select(e => GetDifferenceInYears(e.EndDate, e.StartDate)).Sum();
+
+            if (exp == null)
+                return false;
+
+            if (MALE.Equals(person.Sex, StringComparison.OrdinalIgnoreCase) && exp < _settings.MinExpMale || exp < _settings.MinExpFemale)
```

Comment:
`( )` missing for experience checks, behavior can be wrong.

Reply:
That should work without `( )`.